assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import {
+  useMinischeduleBlock,
+  useMinischeduleRun,
+} from "../../hooks/useMinischedule"
+import { Block, Run } from "../../minischedule"
+import { TripId } from "../../schedule"
+import Loading from "../loading"
+
+export interface Props {
+  activeTripId: TripId
+}
+export const MinischeduleRun = ({ activeTripId }: Props) => {
+  const run: Run | null = useMinischeduleRun(activeTripId)
```

Comment:
Waiting to test these two until we have actual content there seems fine by me.

Reply:
I ended up adding tests, since the loading/null/data conditionals are reasonably permanent, even if the content of the snapshot will change soon.